### DISCUSSION TITLE
Password interface input?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15975] [Archive]
Apart the password used to access the app, we need to store password with argon2 hash.
Is there a ready made input with Password interface or do I have to build an interface extension?

**Comment by br41nslug** [comment]
The "Hash" interface allows you to store argon2 hashed secrets 
![image](https://user-images.githubusercontent.com/9389634/195540987-e2603e1c-f7ef-4c29-8447-704a5c01de59.png)

**Reply by chris-saunders-rocketlab** [reply]
I need to reset a users password outside of the built in forgot password method.
I tried this in a flow but it didn't work.
I added a hash field to directus_users, updated the user by putting the password in that field, then read the user again to get the hashed value and update again sending the hashed value as the password.
The user cannot log in after this process.
Below are the configs and payloads for the relevant part of the flow.
Anyone know if this method should work or what may be wrong?
<img width="503" alt="image" src="https://user-images.githubusercontent.com/74514057/216492105-b8c09a90-c133-4f1b-bc81-7c758a4882f4.png">
<img width="747" alt="image" src="https://user-images.githubusercontent.com/74514057/216492235-f75c6e3e-2b76-45bf-996e-d7c58316338d.png">
<img width="860" alt="image" src="https://user-images.githubusercontent.com/74514057/216492438-d0c1019b-ecde-4298-bded-bddc304ca184.png">
<img width="861" alt="image" src="https://user-images.githubusercontent.com/74514057/216492541-5ea1c5f5-1f5a-4368-ae2a-41a1df631400.png">
